.knip.jsonc

```diff
@@ -3,29 +3,18 @@
   "workspaces": {
     ".": {
       "ignoreBinaries": ["only-allow"],
-      "ignoreDependencies": [
```

Comment:
This is reformatted because master isn't formatted fully; I'll send a PR to deal with that.

Reply:
https://github.com/microsoft/DefinitelyTyped-tools/pull/802